applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_client.c

```diff
@@ -1533,34 +1436,43 @@ int unicast_client_discover(struct bt_conn *conn, enum unicast_discover_dir dir)
 	return ret;
 }
 
-int unicast_client_start(void)
+int unicast_client_start(enum bt_audio_dir dir)
 {
-	int ret_left = 0;
-	int ret_right = 0;
+	int ret;
+	int ret_total = 0;
```

Comment:
```suggestion
	int err_total = 0;
```
We normally use ret, but at this point we are sure some error has ocurred.